src/test/java/com/autonomic/tmc/auth/ClientCredentialsTokenSupplierTest.java

```diff
@@ -41,7 +41,6 @@
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Stream;
 import lombok.extern.slf4j.Slf4j;
-import org.apache.log4j.BasicConfigurator;
```

Comment:
**This comment is NOT related to this line or the changes in scope of this PR**

I noticed the following WARNING in Maven output:
```
[WARNING] /Users/jenishshah/code/autonomic-tmc/tmc-auth/src/test/java/com/autonomic/tmc/auth/ClientCredentialsTokenSupplierTest.java: /Users/jenishshah/code/autonomic-tmc/tmc-auth/src/test/java/com/autonomic/tmc/auth/ClientCredentialsTokenSupplierTest.java uses unchecked or unsafe operations.
```

The fix for this Warning is to change method signature as follows:

```java
    <T extends Throwable> void anyExceptionThrown_alwaysRethrowsAsSdkException(String expectedMessage, Throwable cause,
        Class<T> expectedExceptionOfType) {
```